tests/test_sql_magic.py

```diff
@@ -3,48 +3,6 @@
 
 import pandas as pd
 import pytest
-from IPython.core.interactiveshell import InteractiveShell
-
-from noteable_magics.sql.connection import Connection
-from noteable_magics.sql.magic import SqlMagic
-
-
```

Comment:
All this stuff was moved up to main conftest.py level a few PRs ago. Meant to delete from here also.